_posts/2020-07-16-omero-5-6-2.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: post
+title: Release of OMERO.server 5.6.2 and OMERO.web 5.7.0
```

Comment:
:+1: One could imaging including OMERO.iviewer as the latest release consume API additions from OMERO.web 5.7.0. Has this been discussed?

Reply:
yes, I discussed with @will-moore and he has prepared a PR https://github.com/ome/omero-iviewer/pull/331

Reply:
Do we want to combine all the release in one announcement?

Reply:
Given that OMERO.iviewer 0.10.0 requires OMERO.web 5.7.0 I think it makes sense to combine them.